cpp/distributed/dist-mnist.cpp

```diff
@@ -192,5 +182,5 @@ int main(int argc, char* argv[]) {
               << std::endl;
   } // end rank 0
 
-  MPI_Finalize();
+  mpiExit();
```

Comment:
Not sure if this is the right way to finalize the MPI environment?

Reply:
`mpiExit` is called for you, you don't need to call it separately.

Reply:
Removed it